modules/azure/api_management_api_operation/variables.tf

```
@@ -41,6 +41,7 @@ variable "description" {
 variable "request_content_type" {
   type        = string
   description = "Request representation content type"
+  default     = ""
```

Is this not the correct way?
```suggestion
  default     = null
```
https://github.com/recognizegroup/terraform/pull/400/files#diff-b61ab0285e3ba292b3f3c6d575413cc52f391eea05792549e9a723e05218aedfR33
```terraform
for_each = var.request_content_type != null ? tolist([var.request_content_type]) : []
```
would always result in `var.request_content_type != null` being true so you always get `tolist([var.request_content_type])` and never `[]`


or an other option is to not check for null but for empty string like:
```terraform
for_each = var.request_content_type != "" ? tolist([var.request_content_type]) : []
```

in this option you keep the default value `""`

this second option may be even beter.

---

You are right!

changed to following 
`for_each = var.request_content_type != "" ? tolist([var.request_content_type]) : []`